tests/tck/features/match/WithUnwind.feature

```diff
@@ -186,3 +186,36 @@ Feature: With clause and Unwind clause
       | "hello" | 1 | 1 |
       | "hello" | 2 | 2 |
       | "hello" | 3 | 3 |
+
+  Scenario: match with return
+    When executing query:
+      """
+      MATCH (v:player)
+      WITH v.age AS age, v AS v, v.name AS name
+         ORDER BY age DESCENDING, name ASCENDING
+         LIMIT 20
+         WHERE age > 30
```

Comment:
Is there order requirement of order by and where ? I think filter first then sort is equivalent.

Reply:
The where clause in the with clause must be executed last, which is specified by the openCypher standard. In fact, the where clause here is more like the having clause in sql.
You can refer to [https://neo4j.com/docs/cypher-manual/current/clauses/with/#with-filter-on-aggregate-function-results](https://neo4j.com/docs/cypher-manual/current/clauses/with/#with-filter-on-aggregate-function-results) for details.